src/core/Table/Table.test.tsx

```diff
@@ -1043,3 +1043,24 @@ it('should pass custom props to row', () => {
   expect(onMouseEnter).toHaveBeenCalledWith(mockedData()[0]);
   expect(element).toBeTruthy();
 });
+
+it('should render condensed table', () => {
```

Comment:
You could use `it.each` here like in `Text.test.tsx`:
![image](https://user-images.githubusercontent.com/36186912/129535376-6fab658d-2d65-40d7-9d7b-20470463636d.png)